src/app/components/WorkDetail/DefinitionList.jsx

```diff
@@ -52,117 +51,62 @@ export const DefinitionList = ({ eReaderUrl, data }) => {
-                  {htmlEntities.decode(entity.name)}, {entity.roles.join(', ')}
-                </Link>
-                {birthDate(entity)}
-                {entity.viaf && (
-                  <a
-                    target="_blank"
-                    href={`https://viaf.org/viaf/${entity.viaf}`}
-                    rel="noopener noreferrer"
-                  >
-                    (viaf)
-                  </a>
-                )}
-                {entity.lcnaf && (
-                  <a
-                    target="_blank"
-                    href={`http://id.loc.gov/authorities/names/${entity.lcnaf}.html`}
-                    rel="noopener noreferrer"
-                  >
-                    (lcnaf)
-                  </a>
-                )}
-              </li>
-            ))}
-          </ul>
-        );
+        return <AuthorsList agents={list} />;
 
       case 'subjects':
         return (
           <ul>
             {list.map((subject, i) => (
-              <li key={i.toString()}>
+              <li key={`subject${i.toString()}`}>
                 <Link
                   to={{
                     pathname: '/search',
                     query: { q: `"${subject.subject}"`, field: 'subject' },
                   }}
                 >
-                  {Html5Entities.decode(subject.subject)}
+                  {htmlEntities.decode(subject.subject)}
                 </Link>
               </li>
             ))}
           </ul>
         );
-
-      case 'instances':
+      case 'identifiers':
         return (
-          <table className="nypl-basic-table">
-            <thead>
-              <tr>
-                <th>eBooks</th>
-                <th>Date of Publication</th>
-                <th>Place of Publication</th>
-                <th>Publisher</th>
-              </tr>
-            </thead>
-            <tbody>
-              {list.map((instance, i) => {
-                const isValid =
-                  (instance.items && instance.items.length > 0) ||
-                  (instance.pub_date && instance.pub_date_display) ||
-                  instance.pub_place ||
-                  instance.publisher;
-                if (!isValid) {
-                  return null;
-                }
-                return (
-                  <tr key={i.toString()}>
-                    <td>
-                      {instance.items ? (
-                        <EBookList ebooks={instance.items} eReaderUrl={eReaderUrl} />
-                      ) : (
-                        ''
-                      )}
-                    </td>
-                    <td>{instance.pub_date ? instance.pub_date_display : ''} </td>
-                    <td>{instance.pub_place ? `${instance.pub_place}` : ''}</td>
-                    <td>{instance.publisher}</td>
-                  </tr>
-                );
-              })}
-            </tbody>
-          </table>
+          <ul className="inline-list">
```

Comment:
Prefix the className here with nypl- or similar. We need to establish a convention.